### PR TITLE
Backport refactor for transactions partials

### PR DESCRIPTION
**Pull request by maria-robobug**
- Backports changes from this PR  https://github.com/couchbase/docs-sdk-common/pull/45 
- Changes xrefs for 7.0 to **7.1**

### CODE REVIEW

File: modules/shared/partials/acid-transactions.adoc [hunk 1]
```diff
@@ -206,28 +217,29 @@ Note that expiration is not guaranteed to be followed precisely.
...
 
 As discussed <<mechanics,previously>>, each transaction has a 'single point of truth' that is updated atomically to reflect whether it is committed.
 
 However, it is not always possible for the protocol to become 100% certain that the operation was successful, before the transaction expires.
 That is, the operation may have successfully completed on the cluster, or may succeed soon, but the protocol is unable to determine this (whether due to transient network failure or other reason).
 This is important as the transaction may or may not have reached the commit point, e.g. succeeded or failed.
 
-The library raises TransactionCommitAmbiguous to indicate this state.
-It should be rare to receive this exception.
+Couchbase transactions will raise `{transaction-commit-ambiguous}` to indicate this state.
```
Comment (osfameron) [inline modules/shared/partials/acid-transactions.adoc:228]: the code backticks need to be within the variable don't they?

Comment (osfameron) [reply modules/shared/partials/acid-transactions.adoc:228]: (same in a few places. But I see e.g. `{ctx-get}` does it with the backticks inside the variable, which seems correct?)

**Review by osfameron — CHANGES_REQUESTED** [review]
🤔 Is it a "Backport" from 7.0 to 7.1 ?

Changes/clarification requested on backticks with variable expansions in them, hence "Request changes"

File: modules/shared/partials/acid-transactions.adoc (hunk 1 above)
Comment (maria-robobug) [reply modules/shared/partials/acid-transactions.adoc:228]: Not in all cases, for example we use **{transaction-commit-ambiguous}** as a heading as well, so including the back ticks wouldn't allow us to reuse this variable in other places.

https://github.com/couchbase/docs-sdk-common/pull/46/files#diff-99180d5d8fb133a299c04b915f1fac0c35a880d99bca85864460b707ebb8aa6cR220

Comment (osfameron) [reply modules/shared/partials/acid-transactions.adoc:228]: OK! Yup, seems to work, which, honestly surprises me... I'll have to go re-read the asciidoc docs...

In that case, I'm just a little confused about why some of the variables include the code formatting and others don't, but if it works, then 👍

**Review by osfameron — APPROVED** [review]

**Pull request merged**